stylelint.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
 
   plugins: ['stylelint-scss', 'stylelint-order'],
   rules: {
+    'max-line-length': 150,
```

Comment:
yeah, We have a new discussion about removing this rule from the style files since our tokens tend to grow. So far I'm going to keep this value as `null`.